models/cart.model.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const CartSchema = new mongoose.Schema({
```

Comment:
you've already extracted out Schema in line no 2 (`const { Schema } = mongoose;`) , so you can directly create schema using `const CartSchema = new Schema({})`